packages/tabs/src/Tab.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export interface TabProps {
+  /**
+   * Content that will appear as the title in the Tab list.
+   */
+  name: string | React.ReactNode;
+
+  /**
+   * Content that will appear inside of Tab panel.
+   */
+  children?: React.ReactNode;
+
+  /**
+   * Boolean that determines if the Tab is disabled.
+   */
+  disabled?: boolean;
+
+  /**
+   * If Tabs component is uncontrolled, this determines what Tab will be active on first render.
+   */
+  default?: boolean;
+
+  /**
+   * Adds a className to the root element.
+   */
+  className?: string;
+
+  /**
+   * Destination when name is rendered as `a` tag.
+   */
+  href?: string;
+
+  /**
+   * Destination when name is rendered as `Link` tag.
+   */
+  to?: string;
+
```

Comment:
We should rename active to selected to align with the Tabs prop.

Reply:
👍